frontend/src/pages/modelCatalog/screens/ModelDetailsPage.tsx

```diff
@@ -113,33 +182,12 @@ const ModelDetailsPage: React.FC = conditionalArea(
       errorMessage="Unable to load model catalog"
       provideChildrenPadding
       headerAction={
-        loaded &&
-        (modelRegistryServices.length === 0 ? (
-          <Popover
-            headerContent="Register to model registry?"
-            triggerAction="hover"
-            data-testid="register-catalog-model-popover"
-            bodyContent={
-              <PopoverListContent
-                data-testid="Register-model-button-popover"
-                leadText="To request access to the model registry, contact your administrator."
-                listHeading="Your administrator might be:"
-                listItems={FindAdministratorOptions}
-              />
-            }
-          >
-            <Button data-testid="register-model-button" isAriaDisabled>
-              Register model
-            </Button>
-          </Popover>
-        ) : (
-          <Button
-            data-testid="register-model-button"
-            onClick={() => navigate(registerCatalogModel(params))}
-          >
-            Register model
-          </Button>
-        ))
+        loaded && (
+          <ActionList>
+            {tuningAvailable && fineTuneActionItem}
+            {registerModelButton()}
+          </ActionList>
```

Comment:
The spacing between these buttons is more than I would expect. Looking at the [action list PF docs](https://www.patternfly.org/components/action-list#action-list-single-group), I think we need an `<ActionListGroup>` in here even though there's only one group, and then maybe `<ActionListItem>`s around the buttons (when I edit that docs example to have the buttons as direct children of the ActionList I get this same extra spacing). We probably want that for both of the ActionLists here (this one and the one in the popover).